diff.py

```diff
@@ -2050,9 +2078,21 @@ def normalize_stack(row: str, arch: ArchSettings) -> str:
     return re.sub(arch.re_sprel, "addr(sp)", row)
 
 
-def imm_matches_everything(row: str, arch: ArchSettings) -> bool:
-    # (this should probably be arch-specific)
-    return "(." in row
+def imm_matches_everything(field: str, arch: ArchSettings) -> bool:
+    if arch.name == "ppc":
```

Comment:
can you make this a method on AsmProcessorPPC instead?

Reply:
you mean like a static method?
the processor class object is not available everywhere this is called